sos/report/plugins/firewall_tables.py

```diff
@@ -0,0 +1,79 @@
+from sos.report.plugins import (Plugin, IndependentPlugin, SoSPredicate)
+
+
+class firewall_tables(Plugin, IndependentPlugin):
```

Comment:
Since we already have a `firewalld` plugin, and this PR is adding a `ufw` plugin, I think we should should name this new plugin simply `iptables`, to keep consistent with the component <-> plugin naming.

Reply:
Including nftables in iptables seemed a bit odd to me, but if you think it won't be confusing, happy to do that.

Reply:
That's a fair point. Part of my hesitation with the use of `firewall_tables` is we (will) have 2 other firewall plugins. Rock and a hard place with this naming it seems. I'm open for other suggestions as well of course.

`nftables` is supposed to outright replace `iptables` though, if I'm not mistaken. That, coupled with the fact that pretty much any support engineer or sysadmin is going to know the name `iptables` and its context (but the same can't be necessarily said for `nftables`, is what makes me lean towards naming it `iptables`.

@pmoravec @bmr-cymru thoughts?

Reply:
I lean towards `firewall_tables`, as that is technically accurate while `[ip|nf]tables` is / will be confusing. But I deal the name purely from the technical perspective, hard to say what is the first word a user comes up when thinking about firewall (now or in next few years)..

Reply:
Works for me then. Ack to the name.